Source/ASRunLoopQueue.mm

```diff
@@ -342,22 +352,15 @@ @implementation ASCATransactionQueue
 // but after most other scheduled work on the runloop has processed.
 static int const kASASCATransactionQueueOrder = 1000000;
 
-+ (ASCATransactionQueue *)sharedQueue NS_RETURNS_RETAINED
-{
-  static dispatch_once_t onceToken;
-  static ASCATransactionQueue *sharedQueue;
-  dispatch_once(&onceToken, ^{
-    sharedQueue = [[ASCATransactionQueue alloc] init];
-  });
-  return sharedQueue;
-}
-
 - (instancetype)init
 {
   if (self = [super init]) {
-    _runLoop = CFRunLoopGetMain();
-    NSPointerFunctionsOptions options = NSPointerFunctionsStrongMemory;
-    _internalQueue = [[NSPointerArray alloc] initWithOptions:options];
+    _internalQueueHashSet = CFSetCreateMutable(NULL, 0, NULL);
+    
+    // This is going to be a very busy queue – every node in the preload range will enter this queue.
+    // Save some time on first render by reserving space up front.
+    _internalQueue.reserve(64);
```

Comment:
We could remove the magic number in here to make it more obvious.

Reply:
Done.